akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1933,6 +1933,22 @@ trait FlowOpsMat[+Out, +Mat] extends FlowOps[Out, Mat] {
     viaMat(GraphStages.terminationWatcher)(matF)
 
   /**
+    * Materializes to `FlowMonitor[Out]` that allows monitoring of the the current flow. All events are propagated
+    * by the monitor unchanged. Note that the monitor inserts a memory barrier every time it processes an
+    * event, and may therefor affect performance.
+    */
+  def monitor: ReprMat[Out, FlowMonitor[Out]] = monitorMat()(Keep.right)
```

Comment:
As far as I can tell from the code, there is only watchTermination, and it behaves like the mat version (i.e.: takes a combiner function)